inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/KnowledgeBaseDetailsPanel.java

```diff
@@ -389,6 +390,16 @@ protected void setUpCommonComponents(WebMarkupContainer wmc) {
 
                 private static final long serialVersionUID = -2101263555896964046L;
 
+                @Override
+                protected void onConfigure()
+                {
+                    setEnabled(false);
```

Comment:
Use `component.add(LambdaBehavior.onConfigure(it -> it.setEnabled(false)));`

But why is it *always* disabled?